### PR TITLE
added small tokenizer to prevent tokenization of emails and urls

### PR DESCRIPTION
**Pull request by mariagardbraga**
Added function small_tokenizer in order to get the tokens of regions correctly computed. Tokenizes punctuation but prevents emails and urls to be tokenized.

### CODE REVIEW

File: unbabel_text_utils/tags_align/align_wa/tags_region.py [hunk 1]
```diff
@@ -53,6 +50,48 @@ def compute_str_to_word(list_tuples_tags, tokens, aux, laux, tags):
...
+
+
+class FootnoteNumbers(object):
+    def __init__(self, str_type='', start=1):
+        self.count = start - 1
+        self.str_type = str_type
+
+    def __call__(self, match):
+        self.count += 1
+        return "{}".format(self.count) + self.str_type
```
Comment (vascop) [inline unbabel_text_utils/tags_align/align_wa/tags_region.py]: I'm not sure the string "<integer>email" will never show up in the text, can we make it something a bit more strange? Like using a special character like |||1email|||?

**Comment by vascop** [comment]
Other than that small comment looks good to go

File: unbabel_text_utils/tags_align/align_wa/tags_region.py (hunk 1 above)
Comment (mariagardbraga) [inline unbabel_text_utils/tags_align/align_wa/tags_region.py]: We cannot use symbols, else they will be tokenized. It can be like zzz1emailzzz?

**Comment by mariagardbraga** [comment]
@vascop this might be strange enough.

**Pull request merged**